porting/first-steps.rst

```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc lzop imagemagick libncurses5 rsync
 
+.. Note::
+    repo is not available in repositories of Ubuntu 20.04 or later versions. Hence it has to be installed manually.
```

Comment:
Let's make it more explicit:

The ``repo`` command is not ... 

that's double backticks around the word repo

Reply:
Corrected as part of next commit